CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[8.0.0]](https://github.com/iExecBlockchainComputing/iexec-sms/releases/tag/v8.0.0) 2023
+
+* Add iExec banner at startup.
+* Support SMS in enclave for Scone TEE tasks.
+* Upgrade to Spring Boot 2.6.14.
+* Refactor secret model.
+* Upgrade to Gradle 7.6.
+* Update web3 secret challenge.
+* Remove TLS context on server.
+* Remove `GET /secrets` endpoints.
+* Support Gramine framework for TEE tasks.
+* Add `GET /up` client method in iexec-sms-library.
+* Remove non-TEE workflow.
+* Return a same `SmsClient` from the `SmsClientProvider` of iexec-sms-library when calling a same SMS URL.
+* Improve code quality.
```

Comment:
Could we define an order for changelog's lines? We can have something like:
- Dependencies updates
- New features
- Bug fixes
- Code quality (= refactor, documentation, ...)